GoogleForCreators/web-stories-wp
# Checklist: Warn about first page animations

## Context


Add a checklist item for 'First Page Animation' if there are animations on the first page to alert the user about the AMP spec not allowing animations on the first page

## Summary


Adds 'First Page Animation' check to the checklist

## Relevant Technical Choices


The AMP spec doesn't allow animations on the first page of stories for performance reasons, we need to alert the user about this so they know why the animations aren't playing on the first page of their story.

## To-do


UX question: https://github.com/googleforcreators/web-stories-wp/issues/11320#issuecomment-1118887914
## User-facing changes


Before/After
![2022-05-06_11-06-23](https://user-images.githubusercontent.com/14057928/167160514-32d99753-6cd8-4f61-b14b-a8e206e7d0fe.png)

## Testing Instructions






This PR can be tested by following these steps:

1. Go to the Story Editor
2. Create a page with an animation on it - Go to any page other than the first page add an element (Text, Shape, etc) then apply an animation to it
3. Move the page with an animation on it (created in step two) to the first page, using the page carousel at the bottom or deleting all the other pages
4. Open the 'Checklist' in the bottom navigation to the right of the page carousel

On Staging:
- Notice there is not checklist item for 'First Page Animation'

On Branch:
- Notice there is now a checklist item for 'First Page Animations' alerting the user why the animations on the first page are not allowed/seen

## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #11320
